lms/templates/instructor/instructor_dashboard_2/student_admin.html

```diff
@@ -232,7 +238,7 @@ <h5 class="hd hd-5">${_("Task Status")}</h5>
 %endif
 
 %if settings.FEATURES.get('ENABLE_INSTRUCTOR_BACKGROUND_TASKS'):
-<div class="running-tasks-container action-type-container">
+<div class="running-tasks-container action-type-container ${'hidden' if gradebook_related_sections_hidden else ''}">
```

Comment:
We could just exclude this section (and others) conditionally instead of adding the `hidden` class, but this was breaking [this code](https://github.com/openedx/edx-platform/blob/open-release/palm.4/lms/static/js/instructor_dashboard/util.js).